src/components/FormulaEditor.jsx

```diff
@@ -44,6 +44,11 @@ const styles = {
             border: `1px solid ${main}`
         }   
     },
+    disabled: {
+        '&:hover': {
```

Comment:
azt nem tudom, hogy a border milyen színű legyen ilyen esetben. A Dani ábráin szerintem világosabb

Reply:
Egyeztettem Danival, ő is azt mondta, hogy jó lesz ez a szín.